src/index.html

```diff
@@ -1,13 +1,240 @@
+               alt="img">
+            <p class="main__name--text">
+               BEOSOUND THEATRE
+            </p>
+            <p class="main__text">
+               Four patent-pending acoustic innovations in one sleek soundbar envelop you in sound from every angle. Made to fit any TV, and last for generations.
+            </p>
+            <p class="main__it-costs">
+               $ 3199.00
+            </p>
+          </div>
+        </div>
+      </section>
+      <section class="main__categories" id="categories">
+        <h2 class="main__title">
+          Sound. Vision.
+          <br>
+           It’s your pick.
+        </h2>
+        <div class="main__categories__item">
+
+          <img src="../src/images/main__images/categories/speaker-1.svg" class="image image__categories"
+            alt="img">
+          <img src="../src/images/main__images/categories/speaker-2.svg" class="image image__categories"
+            alt="img">
+
+          <p class="main__name--text">
+            Portable speakers
+          </p>
+        </div>
+
+        <img src="../src/images/main__images/categories/earphones-1.svg" class="image image__categories"
+          alt="img">
+        <img src="../src/images/main__images/categories/earphones-2.svg" class="image image__categories"
+          alt="img">
+
+        <p class="main__name--text">
+          Earphones
+        </p>
+
+        <img src="../src/images/main__images/categories/smart-home-1.svg" class="image image__categories"
+          alt="img">
+        <img src="../src/images/main__images/categories/smart-home-2.svg" class="image image__categories"
+          alt="img">
+
+        <p class="main__name--text">
+          Smart Home
+        </p>
+      </section>
+      <p class="main__bottom">
+        View all
+      </p>
+      <section class="about--us" id="about-us">
+        <img src="../src/images/main__images/about--us.svg" class="image__about--us"
+          alt="img">
+        <div class="about--us__content">
+          <h1 class="about--us__title">
+            Timeless, for 50
+            <br> years and
+            <br>counting
+          </h1>
+          <p class="main__text">
+            Bang & Olufsen is expanding its Recreated Classics Program with the launch of Beosystems – a limited edition music system that transcends time by bridging the gap between one of our iconic designs from 1972 and today’s cutting-edge digital technology.
+          </p>
+        </div>
+      </section>
+    </main>
+    <div class="footer-1260px">
+      <section class="contact--us" id="contact-us">
+        <div class="contact--us__content">
+          <p class="contact--us__title">
+            Contact us
+          </p>
+          <form onsubmit="this.submit(); this.reset(); return false;" class="form">
+            <input class="form__element" type="text"
+              id="name" name="name"
+              placeholder="Name">
```

Comment:
```suggestion
            <input 
              class="form__element" 
              type="text"
              id="name" 
              name="name"
              placeholder="Name"
            >
```
Consider looking through your code and fixing similar issues